### ISSUE TITLE
Adding image dialog broken

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#579]
```
Hey there,

hope someone can fix this - I guess it is not a big thing, but with the current 
state the plugin is not usable for me.

What steps will reproduce the problem? (Be *specific*)
1. Add an image field to a custom content type
2. Try to add an image via "choose image" button 
3. Close the dialog

What is the expected output?

1. nice looking image dialog

2. when I close the dialog, everything looks like it looked before opening


What do you see instead?

1. I think the image-choosing dialog should be looking differently, for me it 
looks broken (see screenshot I). 

2. When I close the dialog again, my scrollbar is gone. This is because the 
<body> element still has the class "modal-open" to it, which sets overflow to 
'hidden' for the body. So I cannot scroll the page any longer (without fiddling 
in the chrome inspector, which is not cool).


Does the problem continue if you disable all other plugins? (plugin
conflicts cause a LOT of problems, so please isolate the problem)

--> jup, still broken after I disable all other plugins. Dialog looks messed 
up, cannot scroll after closing


Please copy and paste the system information that appears in a yellow
textarea when you click on "Report a Bug" on any CCTM admin screen (this
includes the version of the
plugin, the version of PHP, the version of MySQL, a list of other active
plugins etc.):

*SYSTEM INFO* 
------------------------ 
Plugin Version: 0.9.7.13-pl
WordPress Version: 4.0
PHP Version: 5.5.9-1ubuntu4.4
MySQL Version: 5.5.38-0ubuntu0.14.04.1
Server OS: Linux
Language: WPLANG
------------------------ 
ACTIVE PLUGINS: 
 * Custom Content Type Manager : Advanced Custom Post Types v.0.9.7.13 [http://code.google.com/p/wordpress-custom-content-type-manager/]
------------------------ 
CURRENT THEME: Twenty Twelve v.1.5 http://wordpress.org/themes/twentytwelve

==================================================

Really really hope some of you guys can help me out on this! Love the plugin, 
but without being able to add images it is somewhat useless for me right now. 
Thanks a lot in advance!

```

Original issue reported on code.google.com by `st.ploes...@gmail.com` on 30 Oct 2014 at 11:23

Attachments:
- [screenshot_bug_I.png](https://storage.googleapis.com/google-code-attachments/wordpress-custom-content-type-manager/issue-579/comment-0/screenshot_bug_I.png)
- [screenshot_bug_II.png](https://storage.googleapis.com/google-code-attachments/wordpress-custom-content-type-manager/issue-579/comment-0/screenshot_bug_II.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `ever...@fireproofsocks.com` on 30 Oct 2014 at 12:20
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
Me too...  :)

A user has been complaining about this for a while, but they described it as 
the page "crashing", so I had been assuming it was a javascript error that was 
overloading the browser, and was hoping it would get fixed by changing 
browsers, or an upgrade or something.

But, I finally looked into it today, and it is definitely a bug, and is exactly 
as this reporter describes.

I haven't looked into it enough to see the modal class causing the issue, but 
it is the weirdest thing - the scroll bar is gone, so I can use page-up and 
page-down, or tabbing between controls to get the page to scroll, but can't 
scroll with my mouse...

Since this reporter says it is a CSS issue, it sounds like a simple bug.  Are 
you able to take a look at it to fix it?  Or does it not break for you, I can 
look for more details if you need them.

Thanks.
```

Original comment by `jonda...@gmail.com` on 13 Apr 2015 at 1:53

**Comment by GoogleCodeExporter** [comment]
```
Ah, there are other bug reports as well:

and these have solutions:

https://code.google.com/p/wordpress-custom-content-type-manager/issues/detail?id
=572
https://code.google.com/p/wordpress-custom-content-type-manager/issues/detail?id
=573

```

Original comment by `jonda...@gmail.com` on 13 Apr 2015 at 2:01

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed with CSS updates released in 0.9.8.x
```

Original comment by `ever...@fireproofsocks.com` on 11 May 2015 at 5:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `ever...@fireproofsocks.com` on 11 May 2015 at 5:42
- Changed state: **Fixed**
